include/enums.h

```diff
@@ -1162,6 +1162,13 @@ enum Phases {
     PHASE_PLAYER_HAPPY              = 30,
 };
 
+enum ActorClasses {
+    ACTOR_CLASS_PLAYER      = 0x000,
+    ACTOR_CLASS_PARTNER     = 0x100,
+    ACTOR_CLASS_ENEMY       = 0x200,
+    ACTOR_CLASS_MASK        = 0x700,
+};
```

Comment:
Can you remove the enum with ENEMY0 etc

Reply:
we still need the other enum to refer to actors by their IDs. SELF is especially important, but you'll find battle sections (mostly bosses) with AI tied to hard-coded actor IDs.